.script/tests/asimParsersTest/runAsimTestersOnForkedRepo.ps1

```diff
@@ -0,0 +1,97 @@
+Param([string]$fork, [string]$branch, [string]$repoBaseFolder)
+
+$global:failed=0
+$global:subscriptionId="419581d6-4853-49bd-83b6-d94bb8a77887"
+$global:workspaceId="059f037c-1b3b-42b1-bb90-e340e8c3142c"
+$global:schemas = ("DNS", "WebSession", "NetworkSession", "ProcessEvent")
+
+
+function run {
+    $subscription = Select-AzSubscription -SubscriptionId $global:subscriptionId
+    $modifiedSchemas = & "$($PSScriptRoot)/../../getModifiedASimSchemas.ps1"
+    $modifiedSchemas | ForEach-Object { testSchema($_) }
+}
+
+function testSchema([string] $fork, [string] $branch, [string] $repoBaseFolder) {
+    if ($repoBaseFolder) {
+        Set-Location $repoBaseFolder
+    } else {
+        $repoBaseFolder = "$($PSScriptRoot)/../../../"
+    }
+
+	$filesThatWereChanged=$(echo $(git diff --name-only))
+	if($filesThatWereChanged) {
+		Write-Error "Please commit your changes or stash them before run the script. "
+	}
+	else {   
+        Write-Host "git remote add $($fork) https://github.com/$($fork)/Azure-Sentinel"
+		git remote add $fork "https://github.com/$($fork)/Azure-Sentinel"
+
+        Write-Host "git fetch $($fork)"
+		git fetch $fork
+
+        Write-Host "git checkout $($branch)"
+		git checkout $branch
+
+        Write-Host "runAsimTesters"
+        & "$($repoBaseFolder)/.script/tests/asimParsersTest/runAsimTesters.ps1"
+	}
+}
+
+function testParser([Parser] $parser) {
+    Write-Host "Testing parser- '$($parser.Name)'"
+    $letStatementName = "generated$($parser.Name)"
+    $parserAsletStatement = "let $($letStatementName)= ($(getParameters($parser.Parameters))) { $($parser.OriginalQuery) };"
+
+    Write-Host "-- Running schema test for '$($parser.Name)'"
+    $schemaTest = "$($parserAsletStatement)`r`n$($letStatementName) | getschema | invoke ASimSchemaTester('$($parser.Schema)')"
```

Comment:
It assumes that the testers were deployed to the workspace. We can document it in the instructions I mentioned above. Better would be to read and chain them also as part of the call.